catalogs/catalog-hadoop/src/main/java/org/apache/gravitino/catalog/hadoop/authentication/kerberos/KerberosConfig.java

```diff
@@ -125,5 +139,13 @@ public int getFetchTimeoutSec() {
                   false /* immutable */,
                   60 /* defaultValue */,
                   false /* hidden */))
+          .put(
+              KRB5_CONF_KEY,
+              PropertyEntry.stringOptionalPropertyEntry(
+                  KRB5_CONF_KEY,
+                  "The Kerberos krb file for the catalog",
+                  false /* immutable */,
+                  "/etc/krb5.conf" /* defaultValue */,
```

Comment:
As the default value is `/etc/krb5.conf`, what if the file is not found?

Reply:
If the file is not exist, "Can't get Kerberos realm" error message will be shown. This behavior is same as before.

Reply:
OK